products/appengine/api.yaml

```diff
@@ -515,4 +515,70 @@ objects:
                 Pathname within the host. Must start with a "/". A single "*" can be included at the end of the path.
                 The sum of the lengths of the domain and path may not exceed 100 characters.
               required: true
+  - !ruby/object:Api::Resource
+    name: 'ServiceSplitTraffic'
+    description: |
+      Traffic routing configuration for versions within a single service. Traffic splits define how traffic directed to the service is assigned to versions.
+    base_url: 'apps/{{project}}/services'
+    create_url: 'apps/{{project}}/services/{{service_id}}?migrateTraffic={{migrate_traffic}}&updateMask=split'
+    create_verb: :PATCH
+    update_url: 'apps/{{project}}/services/{{service_id}}?migrateTraffic={{migrate_traffic}}'
+    update_verb: :PATCH
+    update_mask: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      api: 'https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services'
+    async: !ruby/object:Api::Async
```

Comment:
I think all the occurrences of `::Async` here need to be `::OpAsync`, but I'm not sure. Can you amend them and make sure that it compiles locally?

Reply:
hmm... I was able to compile. But let me check after rebase.

Reply:
@rileykarson I went on long vacation, came back.
I have checked in the latest code with the rebase on the current master.
Everything fine but I am still getting the following error:
root@52d8e96b9325:/mm# ./integration_tests.sh
==> Checking source code against gofmt...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google -v -run=TestAccAppEngineServiceSplitTraffic_appEngineServiceSplitTrafficExample -timeout 240m -ldflags="-X=github.com/terraform-providers/terraform-provider-google/version.ProviderVersion=acc"
=== RUN   TestAccAppEngineServiceSplitTraffic_appEngineServiceSplitTrafficExample
=== PAUSE TestAccAppEngineServiceSplitTraffic_appEngineServiceSplitTrafficExample
=== CONT  TestAccAppEngineServiceSplitTraffic_appEngineServiceSplitTrafficExample
--- FAIL: TestAccAppEngineServiceSplitTraffic_appEngineServiceSplitTrafficExample (136.23s)
    testing.go:635: Step 1 error: Resource specified by ResourceName couldn't be found: google_app_engine_service_split_traffic.
FAIL
FAIL    github.com/terraform-providers/terraform-provider-google/google 136.256s
GNUmakefile:19: recipe for target 'testacc' failed
make: *** [testacc] Error 1

Reply:
Replied in a comment below.